lib/slack_rewriter/application/slack_rewriter.ex

```diff
@@ -0,0 +1,88 @@
+defmodule SlackRewriter.Application.SlackRewriter do
+  @moduledoc """
+  SlackRewriter
+  """
+  alias SlackRewriter.Application.UserRegistry
+  alias SlackRewriter.Bridge.Slack
+  alias SlackRewriter.Utils.YoutrackMatcher
+
+  @type response :: {:ok, term()} | {:error, String.t()}
```

Comment:
```suggestion
  @type response :: :ok | {:error, String.t()}
```
mi pare che non ce ne facciamo niente del ritorno (di fatto anzi non c'è un ritorno) a parte sapere che è ok, è bello che i tipi riflettano questa cosa 😉